src/js/events/__tests__/CosmosPackagesActions-test.js

```diff
@@ -910,7 +910,7 @@ describe("CosmosPackagesActions", function() {
   });
 
   describe("#addRepository", function() {
-    it("dispatches the correct action when successful", function(done) {
+    it.skip("dispatches the correct action when successful", function(done) {
```

Comment:
Are these skips intentional?

Reply:
Yes. this commit goes into some detail:
https://github.com/dcos/dcos-ui/pull/2864/commits/7cc385e430709676416809897b41b9304ae38886

This tests break because of the way the streams work now, so the tests needs fixing. But fixing the tests was not possible in under 1h. Since we are removing the ComosPackageActions related to fetch and edit packages (they are now streams and we will get rid of them on the PR I am working right now) I dont think it pay of to spend more  time fixing them, so they are removed next sprint.

Reply:
okay, makes sense, thank you for the explanation, I should have looked into the commits earlier 🙈

Reply:
No problem 💃